sar-op-sentinel1/src/main/java/eu/esa/sar/sentinel1/gpf/TPGManager.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2024 by SkyWatch Space Applications Inc. http://www.skywatch.com
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 3 of the License, or (at your option)
+ * any later version.
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, see http://www.gnu.org/licenses/
+ */
+package eu.esa.sar.sentinel1.gpf;
+
+import org.esa.snap.core.datamodel.TiePointGrid;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Temporary solution to removing band TPGs from product.
+ */
+public class TPGManager {
+
+    private static TPGManager _instance = null;
+
+    private final Map<String, TiePointGrid> etadTPGMap = new HashMap<>();
```

Comment:
This will keep in memory for the lifetime of the class. We should next consider writing to a file instead